docker-compose.yml

```diff
@@ -12,6 +12,8 @@ volumes:
 services:
   postgres:
     image: postgres:9.6
+    environment:
+      POSTGRES_PASSWORD: notasecret
```

Comment:
We also have to update every app's postgres URL to include this, see https://github.com/alphagov/publishing-e2e-tests/commit/bd8a6b897a3b3cb3d5345028e4f6adf8f1f77cfa

Reply:
Oops - lack of thinking there! I've updated with something (hopefully) better.